x-pack/plugins/observability_solution/apm/server/routes/traces/get_trace_items.ts

```diff
@@ -266,6 +283,34 @@ async function getTraceDocsPerPage({
         },
       },
     },
+    fields: [
```

Comment:
let's not do both _source and `fields`

Reply:
I agree, it's still WIP and currently, I am trying to test everything with `fields` and not use `_source` in the response at all so I will remove the `_source` where the `fields` are used after